src/app/modules/browse/browse.tpl.html

```diff
@@ -16,8 +16,24 @@
                 <h4>Narrow Search</h4>
                 <h5>Languages</h5>
                 <div ng-repeat="lang in topLanguages"><div ng-class="{activeFilter: currentLang == lang.isoCode}"><a class="languageItem" href="" ng-click="filterLanguage(lang.isoCode)" tooltip="{{lang.englishName != lang.name ? lang.englishName : ''}}" tooltip-placement="right" tooltip-trigger="{{'mouseenter'}}">{{lang.name}}</a><a class="clear right" href="" ng-click="filterLanguage('')">Clear</a></div></div>
-                <h5>Tags</h5>
-                <div ng-repeat="tag in topTags"><div ng-class="{activeFilter: currentTag == tag}"><a class="tagItem" href="" ng-click="filterTag(tag)">{{tag}}</a><a class="clear right" href="" ng-click="filterTag('')">Clear</a></div></div>
+                <div ng-show="otherLanguages.length">
+                    <a href="" ng-click="toggleNext($event)">{{otherLanguages.length}} more...</a>
```

Comment:
Do we need to do anything to help localization know that a parameterized message is happening here? That is, the localizer should be invited to translate "{n} more" (languages)